app/decorators/article_decorator.rb

```diff
@@ -1,6 +1,34 @@
 class ArticleDecorator < ApplicationDecorator
   LONG_MARKDOWN_THRESHOLD = 900
 
+  # This method answers whethor or not this decorated article can be
+  # featured in the feed.
+  #
+  # @note From #15292 we want to no longer require the featured
+  #       article to have an image.  However, there are some
+  #       assumptions made in the view about featured articles having
+  #       images.  Furthermore, by enabling a feature flag, if the
+  #       code breaks we can toggle the requirement back on.
+  #
+  # @return [TrueClass] if this article can be "featured" in the feed
+  # @return [FalseClass] if this article should not be "featured" in
+  #         the feed
+  #
+  # @see ./app/views/stories/feeds/show.json.jbuilder for an example
+  #      of usage
+  def can_be_featured_in_feed
+    return false unless featured
+    return true if main_image.present?
+    return true unless FeatureFlag.accessible?(:featured_story_must_have_main_image)
```

Comment:
Can you talk a bit about using `accessible?` here over `enabled?`? I'm not sure I understand the distinction. I looked at [the method itself](https://github.com/forem/forem/blob/58a4d805d77be8ae012d078b7a4fe7b76fbafa58/app/services/feature_flag.rb#L5-L7) and I don't think I understand what it intends to do.

Speculation: it looks like you intend the feature flag to default to "on" in order to avoid confusing negation ("does not need an image when true"), whereas `FeatureFlag.enabled?` defaults to "off". Is that accurate?

Reply:
I definitely want this to default to "on".  And yeah, that method is compact but not quickly mentally parseable.

I'm going to dive into that method and document it and refactor it so it's a bit more "transparent".  I'll get back to you on my spelunking.